src/main/scala/io/delta/tables/execution/DescribeDeltaHistoryCommand.scala

```diff
@@ -36,8 +36,8 @@ case class DescribeDeltaHistoryCommand(
     val basePath =
       new Path(if (table.nonEmpty) {
         DeltaTableIdentifier(sparkSession, table.get) match {
-          case Some(id) if id.path.isDefined => id.path.get
-          case Some(id) => throw DeltaErrors.tableNotSupportedException("DESCRIBE HISTORY")
+          case Some(id) => id.path.getOrElse(
+            throw DeltaErrors.tableNotSupportedException("DESCRIBE HISTORY"))
```

Comment:
we generally put `throw X` in `{ }` so make this, and at the other place `getOrElse { throw ... }`

Reply:
done 👍